### ISSUE TITLE
"Show Table diff" does not compare list/tuple type columns.

### ISSUE THREAD

**Issue by kumarshreshtha** [aimhubio/aim#2361]
## 🐛 Bug
When comparing runs, the "Show table diff" button keeps columns which have list/tuple entries even when the items inside the list are same.

### To reproduce

Run this snippet twice:

```py
import aim

run = aim.Run(experiment="bug_report")
run["hparams"] = {"param1": [0.2, 0.3], "param2": 0.9}

```

Here's the run in the UI:
![image](https://user-images.githubusercontent.com/23312942/202578906-fe2cb62e-b1e4-409b-a646-854c9a4ddfcd.png)

and the result after "Show Table diff":
![image](https://user-images.githubusercontent.com/23312942/202579000-1fbc37ab-c1ab-4514-8a1d-00e7cb502243.png)

### Expected behavior

Should not display `param1` either since it's the same in the two runs.

### Environment

- Aim v3.14.4
- Python 3.8.10
- pip 20.0.2
- Linux

**Comment by roubkar** [comment]
Hey @kumarshreshtha, thanks for reporting this issue.
This functionality is broken in Runs, Params, and Scatters Explorers, but works as expected in Metrics and Images Explorers.
Would you like to open a pull request to resolve this issue?

**Comment by kumarshreshtha** [comment]
yes, on it :)

**Comment by roubkar** [comment]
Hey @kumarshreshtha, your fix has been shipped with `v3.15.0` release.
Please try it out and close this issue if it is resolved.
Congrats on your contribution 🎉.

**Comment by gorarakelyan** [comment]
Closing the issue, as the fix was released.
